[2024-08-31T00:23:25.119+0000] {processor.py:186} INFO - Started process (PID=346) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:23:25.121+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:23:25.136+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:23:25.134+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:23:25.239+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:23:25.229+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:23:25.243+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:23:25.304+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.216 seconds
[2024-08-31T00:23:55.591+0000] {processor.py:186} INFO - Started process (PID=353) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:23:55.593+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:23:55.600+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:23:55.599+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:23:55.640+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:23:55.630+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:23:55.642+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:23:55.675+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.100 seconds
[2024-08-31T00:24:26.036+0000] {processor.py:186} INFO - Started process (PID=361) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:24:26.038+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:24:26.054+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:24:26.044+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:24:26.169+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:24:26.157+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:24:26.173+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:24:26.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.206 seconds
[2024-08-31T00:24:56.649+0000] {processor.py:186} INFO - Started process (PID=367) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:24:56.652+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:24:56.658+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:24:56.657+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:24:56.702+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:24:56.694+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:24:56.704+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:24:56.751+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.116 seconds
[2024-08-31T00:25:27.158+0000] {processor.py:186} INFO - Started process (PID=374) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:25:27.161+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:25:27.166+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:25:27.165+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:25:27.204+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:25:27.197+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:25:27.207+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:25:27.239+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.093 seconds
[2024-08-31T00:25:57.720+0000] {processor.py:186} INFO - Started process (PID=381) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:25:57.723+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:25:57.740+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:25:57.739+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:25:57.812+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:25:57.804+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:25:57.815+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:25:57.854+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.170 seconds
[2024-08-31T00:29:01.842+0000] {processor.py:186} INFO - Started process (PID=38) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:29:01.844+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:29:01.851+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:29:01.850+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:29:01.919+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:29:01.912+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:29:01.923+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:29:01.972+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.148 seconds
[2024-08-31T00:29:32.663+0000] {processor.py:186} INFO - Started process (PID=45) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:29:32.665+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:29:32.671+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:29:32.670+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:29:32.711+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:29:32.703+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:29:32.713+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:29:32.751+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.100 seconds
[2024-08-31T00:30:03.606+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:30:03.614+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:30:03.643+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:30:03.636+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:30:04.078+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:30:04.040+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:30:04.087+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:30:04.490+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.966 seconds
[2024-08-31T00:30:35.051+0000] {processor.py:186} INFO - Started process (PID=62) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:30:35.053+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:30:35.058+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:30:35.057+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:30:35.106+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:30:35.094+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:30:35.109+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:30:35.152+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.117 seconds
[2024-08-31T00:31:05.526+0000] {processor.py:186} INFO - Started process (PID=69) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:31:05.529+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:31:05.534+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:31:05.533+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:31:05.594+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:31:05.587+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:31:05.597+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:31:05.644+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.135 seconds
[2024-08-31T00:31:36.108+0000] {processor.py:186} INFO - Started process (PID=76) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:31:36.111+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:31:36.118+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:31:36.117+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:31:36.170+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:31:36.155+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:31:36.174+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:31:36.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.145 seconds
[2024-08-31T00:32:06.648+0000] {processor.py:186} INFO - Started process (PID=83) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:32:06.650+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:32:06.654+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:32:06.654+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:32:06.686+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:32:06.679+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:32:06.689+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:32:06.725+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.088 seconds
[2024-08-31T00:32:37.125+0000] {processor.py:186} INFO - Started process (PID=90) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:32:37.127+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:32:37.133+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:32:37.132+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:32:37.170+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:32:37.163+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:32:37.172+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:32:37.205+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.092 seconds
[2024-08-31T00:33:07.726+0000] {processor.py:186} INFO - Started process (PID=97) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:33:07.728+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:33:07.740+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:33:07.740+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:33:07.777+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:33:07.770+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:33:07.779+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:33:07.812+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.128 seconds
[2024-08-31T00:33:38.189+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:33:38.192+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:33:38.199+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:33:38.198+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:33:38.258+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:33:38.248+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:33:38.261+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:33:38.394+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.225 seconds
[2024-08-31T00:34:08.800+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:34:08.817+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:34:08.825+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:34:08.825+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:34:08.886+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:34:08.875+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:34:08.892+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:34:08.956+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.181 seconds
[2024-08-31T00:34:39.306+0000] {processor.py:186} INFO - Started process (PID=118) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:34:39.309+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:34:39.317+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:34:39.316+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:34:39.377+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:34:39.370+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:34:39.381+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:34:39.423+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.161 seconds
[2024-08-31T00:35:10.054+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:35:10.057+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:35:10.063+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:35:10.062+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:35:10.104+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:35:10.096+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:35:10.106+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:35:10.145+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.115 seconds
[2024-08-31T00:35:40.536+0000] {processor.py:186} INFO - Started process (PID=132) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:35:40.538+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:35:40.546+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:35:40.545+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:35:40.582+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:35:40.576+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:35:40.585+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:35:40.658+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.153 seconds
[2024-08-31T00:36:10.926+0000] {processor.py:186} INFO - Started process (PID=139) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:36:10.964+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:36:11.001+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:36:11.000+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:36:12.458+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:36:12.449+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:36:12.465+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:36:12.537+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 1.651 seconds
[2024-08-31T00:36:43.059+0000] {processor.py:186} INFO - Started process (PID=147) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:36:43.065+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:36:43.079+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:36:43.071+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:36:43.173+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:36:43.147+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:36:43.177+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:36:43.222+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.193 seconds
[2024-08-31T00:37:13.532+0000] {processor.py:186} INFO - Started process (PID=154) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:37:13.535+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:37:13.542+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:37:13.541+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:37:13.586+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:37:13.580+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:37:13.589+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:37:13.627+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.111 seconds
[2024-08-31T00:37:44.012+0000] {processor.py:186} INFO - Started process (PID=161) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:37:44.014+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:37:44.018+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:37:44.018+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:37:44.055+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:37:44.044+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:37:44.059+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:37:44.104+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.105 seconds
[2024-08-31T00:38:14.635+0000] {processor.py:186} INFO - Started process (PID=169) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:38:14.638+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:38:14.643+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:38:14.643+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:38:14.761+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:38:14.748+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:38:14.765+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:38:14.821+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.201 seconds
[2024-08-31T00:38:45.188+0000] {processor.py:186} INFO - Started process (PID=176) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:38:45.191+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:38:45.200+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:38:45.199+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:38:45.319+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:38:45.287+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:38:45.334+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:38:45.442+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.278 seconds
[2024-08-31T00:39:15.869+0000] {processor.py:186} INFO - Started process (PID=183) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:39:15.872+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:39:15.877+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:39:15.876+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:39:15.912+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:39:15.906+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:39:15.914+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:39:15.949+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.094 seconds
[2024-08-31T00:39:46.231+0000] {processor.py:186} INFO - Started process (PID=190) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:39:46.233+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:39:46.242+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:39:46.241+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:39:46.287+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:39:46.280+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:39:46.291+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:39:46.350+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.133 seconds
[2024-08-31T00:40:16.864+0000] {processor.py:186} INFO - Started process (PID=198) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:40:16.866+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:40:16.871+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:40:16.870+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:40:16.910+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:40:16.904+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:40:16.912+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:40:16.955+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.103 seconds
[2024-08-31T00:40:47.293+0000] {processor.py:186} INFO - Started process (PID=206) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:40:47.296+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:40:47.301+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:40:47.301+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:40:47.341+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:40:47.332+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:40:47.345+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:40:47.388+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.107 seconds
[2024-08-31T00:41:17.768+0000] {processor.py:186} INFO - Started process (PID=213) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:41:17.771+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:41:17.776+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:41:17.775+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:41:17.805+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:41:17.799+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:41:17.807+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:41:17.836+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.083 seconds
[2024-08-31T00:41:48.233+0000] {processor.py:186} INFO - Started process (PID=220) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:41:48.236+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:41:48.244+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:41:48.242+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:41:48.294+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:41:48.286+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:41:48.297+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:41:48.346+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.134 seconds
[2024-08-31T00:42:18.714+0000] {processor.py:186} INFO - Started process (PID=227) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:42:18.716+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:42:18.721+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:42:18.720+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:42:18.778+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:42:18.771+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:42:18.781+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:42:18.814+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.117 seconds
[2024-08-31T00:42:49.235+0000] {processor.py:186} INFO - Started process (PID=234) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:42:49.237+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:42:49.242+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:42:49.241+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:42:49.283+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:42:49.275+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:42:49.286+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:42:49.326+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.101 seconds
[2024-08-31T00:43:19.689+0000] {processor.py:186} INFO - Started process (PID=242) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:43:19.691+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:43:19.695+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:43:19.694+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:43:19.724+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:43:19.719+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:43:19.727+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:43:19.758+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.081 seconds
[2024-08-31T00:47:38.645+0000] {processor.py:186} INFO - Started process (PID=38) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:47:38.648+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:47:38.653+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:47:38.652+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:47:38.726+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:47:38.718+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:47:38.730+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:47:38.775+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.151 seconds
[2024-08-31T00:48:09.240+0000] {processor.py:186} INFO - Started process (PID=45) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:48:09.243+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:48:09.250+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:48:09.249+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:48:09.293+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:48:09.285+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:48:09.296+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:48:09.334+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.109 seconds
[2024-08-31T00:48:39.856+0000] {processor.py:186} INFO - Started process (PID=52) to work on /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:48:39.862+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/blockchain_dag.py for tasks to queue
[2024-08-31T00:48:39.880+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:48:39.879+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:48:39.972+0000] {logging_mixin.py:190} INFO - [2024-08-31T00:48:39.963+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/blockchain_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/blockchain_dag.py", line 5, in <module>
    import cryo
ModuleNotFoundError: No module named 'cryo'
[2024-08-31T00:48:39.976+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/blockchain_dag.py
[2024-08-31T00:48:40.034+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/blockchain_dag.py took 0.222 seconds
